src/languages/html.coffee

```diff
@@ -6,41 +6,141 @@ parse5 = require 'parse5'
 ATTRIBUTE_CLASSES = ['#attribute']
 
 COLORS = {
-  'Default': 'cyan',
-  '#comment': 'grey'
-  'a': 'grey',
-  'b': 'teal',
-  'body': 'return',
-  'br': 'command',
-  'button': 'yellow',
-  'center': 'red',
-  'div': 'amber',
-  'document': 'bluegrey',
-  'font': 'value',
-  'form': 'deeporange',
-  'h1': 'teal',
-  'h3': 'indigo',
-  'head': 'cyan',
-  'hr': 'lime',
-  'html': 'amber',
-  'iframe': 'green',
-  'img': 'green',
-  'input': 'brown',
-  'label': 'lightblue',
-  'li': 'pink',
-  'link': 'purple',
-  'marquee': 'command',
-  'meta': 'error',
-  'option': 'control',
-  'p': 'deeppurple',
-  'script': 'orange',
-  'select': 'indigo',
-  'strong': 'yellow',
-  'table': 'lightgreen',
-  'td': 'lightblue',
-  'title': 'green',
-  'tr': 'bluegrey',
-  'ul': 'blue'
+  #Metadata
```

Comment:
Dropdown is organized as a menu for each argument position (for example, if the first argument of a function is a color and the second argument is a number, then you would have a different set of choices for each position).

In HTML, we would want to have a different organizing scheme, since every attribute has the same choices. We'll have to look into dropdown support later and see what's needed for that.